python_modules/dagster/dagster/_daemon/auto_run_reexecution/auto_run_reexecution.py

```diff
@@ -206,3 +193,7 @@ def consume_new_runs_for_automatic_reexecution(
                 run,
                 engine_event_data=EngineEventData(error=error_info),
             )
+            # mark that this run will not be retried so that the tags reflect the state of the system
+            workspace_process_context.instance.add_run_tags(
+                run_id=run.run_id, new_tags={WILL_RETRY_TAG: "false"}
+            )
```

Comment:
this is a little weird / might be confusing, but as long as you can still reconstruct the sequence of events by looking at the event log, it is probably fine

Reply:
yeah i agree that it's weird, especially since it's really the daemon cursoring that makes updating this tag necessary